crates/forge/bin/cmd/test/mod.rs

```diff
@@ -604,6 +612,47 @@ fn list(
     Ok(TestOutcome::new(BTreeMap::new(), false))
 }
 
+/**
+ * Creates table summary header
+ */
+fn create_test_summary_table_header(summary_table: &mut Table) -> Row {
+    summary_table.apply_modifier(UTF8_ROUND_CORNERS);
+    Row::from(vec![
+        Cell::new("Test Suites")
```

Comment:
Consider adding a verbose version of the summary that splits this into two "File Path" and "Contract Name" columns, since users may have the same contract name in multiple files, so that helps disambiguate

Reply:
you mean adding another flag like `--summary-detailed`? Can also include the test file path by default if it makes sense.

Reply:
I'd maybe do this as a `--detailed` flag which requires `--summary` to be enabled